graphrag/api/query.py

```diff
@@ -469,7 +476,7 @@ def _patch_vector_store(
             from graphrag.vector_stores.lancedb import LanceDBVectorStore
 
             community_reports = with_reports
-            collection_name = (
+            container_name = (
                 config.embeddings.vector_store.get("container_name", "default")
```

Comment:
```suggestion
                config.embeddings.vector_store["container_name"]
```

I recommend to not add default values to fallback on in the python code as we plan to switch over to a better config system that will validate/enforce the existence of `container_name` in the config. It will be better and easier to debug if the python code just breaks when `container_name` is not present in the dict.

Reply:
In this case I think I was just aiming for back-compat. Looking up the function a bit I see that we are creating this with defaults though, so that seems sensible. Once we enforce the new model we can take out the default creation entirely.